tests/suites/hup/suite.js

```diff
@@ -231,24 +231,12 @@ const initDUT = async (that, test, target) => {
 
 	test.comment(`DUT ready`);
 
+	// Retrieving journalctl logs 
 	that.teardown.register(async () => {
-		await that.context.get().hup.archiveLogs(that, test, target);
+		await that.context.get().worker.archiveLogs(that.id, that.context.get().link, "journalctl --no-pager --no-hostname --list-boots | awk '{print $1}' | xargs -I{} sh -c 'set -x; journalctl --no-pager --no-hostname -a -b {};");
```

Comment:
```suggestion
		await that.context.get().worker.archiveLogs(that.id, that.context.get().link, "journalctl --no-pager --no-hostname --list-boots | awk '{print $1}' | xargs -I{} sh -c 'set -x; journalctl --no-pager --no-hostname -a -b {};'");
```
We are missing a close tick `'`